transfer-archive/app/services/tranfer_service.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import subprocess
+import sys
+
+from app.services.models import RcloneAzureConfig, RcloneS3Config
+from app.services import rclone_service
+
+logger = logging.getLogger(__name__)
+
+
+def transfer_data(azure_remote: RcloneAzureConfig, s3_remote: RcloneS3Config):
```

Comment:
```suggestion
def transfer_data(azure_remote: RcloneAzureConfig, s3_remote: RcloneS3Config) -> None:
```

Og typo i filnavnet, `tranfer_service` -> `transfer_service`